setup

```diff
@@ -111,24 +111,30 @@ then
         sed -i "s/\(AUTO_SETUP_NET_WIFI_ENABLED\)=.*/\1=1/" /boot/dietpi.txt
         sed -i "s/\(AUTO_SETUP_NET_WIFI_COUNTRY_CODE\)=.*/\1=FR/" /boot/dietpi.txt
         sed -i "s/\(AUTO_SETUP_NET_HOSTNAME\)=.*/\1=Nabaztag/" /boot/dietpi.txt
-        # zram rather than /var/swap (DietPi default):
-        sed -i "s/\(AUTO_SETUP_SWAPFILE_LOCATION\)=.*/\1=zram/" /boot/dietpi.txt
         sed -i "s/\(AUTO_SETUP_HEADLESS\)=.*/\1=1/" /boot/dietpi.txt
         # -1=Nginx rather than -2=Lighttpd (DietPi default):
         sed -i "s/\(AUTO_SETUP_WEB_SERVER_INDEX\)=.*/\1=-1/" /boot/dietpi.txt
         # -4=Daemon + Drift rather than 2=boot + daily (DietPi default):
         sed -i "s/\(CONFIG_NTP_MODE\)=.*/\1=4/" /boot/dietpi.txt
         sed -i "s/\(CONFIG_SERIAL_CONSOLE_ENABLE\)=.*/\1=0/" /boot/dietpi.txt
-    else
-        # assume Rasberry Pi OS distro
+    elif [ -f "/etc/rpi-issue" ]
+    then
+        # Raspberry Pi OS distro
         hostname="raspberrypi"
         # be kind to users:
+        if [ ! -f "/boot/userconf.txt" ]
+        then
+            # make sure 'pi' user has default 'raspberry' password
+            echo 'pi:$y$j9T$9ZU7FXeU7PVHm49rB5Hbe.$MHZMLJfNfXBFbYgGwg/TNjADIRbna.Epxzd/MrqPJM9' > '/boot/userconf.txt'
```

Comment:
We shouldn't put this default password either.

Reply:
Two options then:

- use another secret default password, known only to Nabaztag owners;
- let users cope by themselves with setting up the password.
We may point them to  the [Raspberry PI imager](https://www.raspberrypi.com/software/), which now allows  configuring (at least for Raspberry Pi OS based images) image settings: Wifi setup, SSH setup, user/password.